src/adapters/season_adapters.py

```diff
@@ -6,27 +6,32 @@
 
 # TODO: Refactor using match case (it looks like ufmt doesn't work with that)
 def to_race_status(race_state: str) -> RaceStatus:
-    if race_state == "results":
+    if race_state == "1":
         return RaceStatus.COMPLETED
-    elif race_state == "default":
+    elif race_state == "0":
         return RaceStatus.SCHEDULED
     else:
         return RaceStatus.UNKNOWN
 
 
 def to_race(game_period: dict) -> Race:
     return Race(
-        race_id=game_period["id"],
-        name=game_period["short_name"],
-        starts_at=datetime.strptime(game_period["starts_at"], "%Y-%m-%dT%H:%M:%S.%fZ"),
-        status=to_race_status(game_period["state"]),
+        race_id=game_period["RaceId"],
+        name=game_period["MeetingLocation"],
```

Comment:
Open discussion: what do you think to use the `MeetingName` or `MeetingOfficialName` field?

Here is a simple JSON:
```
[...]
        "MeetingName": "Bahrain Grand Prix",
        "MeetingOfficialName": "FORMULA 1 GULF AIR BAHRAIN GRAND PRIX 2023",
        "MeetingLocation": "Sakhir",
        "CircuitLocation": "Bahrain",
[...]
```

Reply:
Sounds good. I didn't remember which field we used, so `MeetingName` is fine